static_src/components/service_plan.jsx

```diff
@@ -12,6 +12,12 @@ const propTypes = {
   })
 };
 
+// This should be removed when solution is setup
+// for service instancemultiple params
+const CF_CLI_SERVICE_LIST = [
```

Comment:
is there some way we can use a common structure between this one and the one you introduced in this other PR? https://github.com/18F/cg-dashboard/pull/1197/files#diff-49e04869fdff69f2ac117d46c205eaf9R23